### PR TITLE
Properly adds a drill sergeant hat to the warden's locker

### PR DESCRIPTION
**Pull request by VileBeggar**
## Changelog
:cl: Vile Beggar
add: Warden now has an added drill hat in his locker.
/:cl:

## About The Pull Request
Adds a drill sergeant's hat into the warden's locker, wearing it forces you to shout.
Sprites aren't made by me but I saw them gathering dust in the code though.

image

## Why It's Good For The Game
More choice in fashion is good. The warden doesn't have much of a job to do besides looking pretty at the desk and this hat helps him do that better.
Besides, I know we all wanna do some epic roleplay as our favorite drill sergeant: http://i.imgur.com/fU0sbp5.jpg and of course https://i.imgur.com/EyvX0VB.png

### CODE REVIEW

**Comment by OverlyGeneric** [comment]
Very Angry Canadians bans cheaters from Tea Forest Blue

**Comment by SpessClown** [comment]
Holy Jesus, what is that?

**Comment by wesoda25** [comment]
Make every sentence end in PRIVATE

File: code/modules/clothing/head/jobs.dm [hunk 1]
```diff
@@ -145,6 +145,21 @@
 	strip_delay = 60
 	dog_fashion = /datum/dog_fashion/head/warden
 
+/obj/item/clothing/head/warden/drill
+	name = "warden's drill hat"
+	desc = "A special armored campaign hat with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection. Has the letters 'FMJ' enscribed on it's side."
+	icon_state = "wardendrill"
+	item_state= "wardendrill"
+	dog_fashion = null
+	var/datum/mutation/human/yelling
```
Comment (81Denton) [inline code/modules/clothing/head/jobs.dm]: This can be removed
```suggestion
```

File: code/modules/clothing/head/jobs.dm [hunk 2]
```diff
@@ -145,6 +145,21 @@
 	strip_delay = 60
 	dog_fashion = /datum/dog_fashion/head/warden
 
+/obj/item/clothing/head/warden/drill
+	name = "warden's drill hat"
+	desc = "A special armored campaign hat with the security insignia emblazoned on it. Uses reinforced fabric to offer sufficient protection. Has the letters 'FMJ' enscribed on it's side."
+	icon_state = "wardendrill"
+	item_state= "wardendrill"
```
Comment (81Denton) [inline code/modules/clothing/head/jobs.dm]:
```suggestion
	item_state = "wardendrill"
```

**Review by 81Denton — CHANGES_REQUESTED** [review]

File: code/modules/clothing/head/jobs.dm [hunk 3]
```diff
@@ -145,6 +145,21 @@
...
+	icon_state = "wardendrill"
+	item_state= "wardendrill"
+	dog_fashion = null
+	var/datum/mutation/human/yelling
+
+/obj/item/clothing/head/warden/drill/equipped(mob/living/carbon/human/user, slot)
+	..()
+	if(slot == SLOT_HEAD)
+		user.dna.add_mutation(YELLING)
+	else
```
Comment (81Denton) [inline code/modules/clothing/head/jobs.dm]: Replace the "else" part with `dropped(mob/user)`, else it won't remove the mutation if the hat gets destroyed by acid etc.
Krav maga gloves have similar code that you could use.

**Review by 81Denton — CHANGES_REQUESTED** [review]

File: code/modules/clothing/head/jobs.dm (hunk 3 above)
Comment (JJRcop) [reply code/modules/clothing/head/jobs.dm]: To clarify, the entire proc path is required.
```DM
/obj/item/clothing/head/warden/drill/dropped(mob/user)
```

**Comment by HomocidlePancakes** [comment]
We have a jabroni outfit and now a drill sergeant hat. Can we get a Ricardo costume now?

**Comment by nicbn** [comment]
This was removed because no one wore it and will be removed again after someone realises no one is wearing it

Then, another PR will be open to readd it and we will keep feeding the loop

**Review by 81Denton — APPROVED** [review]
Looks good to me

**Pull request merged**